scene/2d/physics_body_2d.cpp

```diff
@@ -38,13 +38,18 @@
 #include "core/templates/rid.h"
 #include "scene/scene_string_names.h"
 
-void PhysicsBody2D::_notification(int p_what) {
-}
-
 void PhysicsBody2D::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("move_and_collide", "rel_vec", "infinite_inertia", "exclude_raycast_shapes", "test_only"), &PhysicsBody2D::_move, DEFVAL(true), DEFVAL(true), DEFVAL(false));
+	ClassDB::bind_method(D_METHOD("test_move", "from", "rel_vec", "infinite_inertia", "exclude_raycast_shapes", "collision"), &PhysicsBody2D::test_move, DEFVAL(true), DEFVAL(true), DEFVAL(Variant()));
+
+	ClassDB::bind_method(D_METHOD("set_safe_margin", "pixels"), &PhysicsBody2D::set_safe_margin);
```

Comment:
I know this kind of is a nitpick, but safe margin exists as a hack because of Bullet. If Bullet is being removed, I think we should probably just get rid of it, and pass the margin as a default parameter of move_and_collide. this way we also pollute the base class less.

Reply:
Since you are going to remove Bullet now, I guess you like to reread this. 

(I assume that decision hasn't been taken at the time)